### DISCUSSION TITLE
How do you bold the title of your page?

### DISCUSSION THREAD

**Discussion by oycyc** [community/community#22939] [Other Feature Feedback, Questions, & Ideas]
<p> Hello.</p>
<p>I was wondering how you could be able to bold the title of a page. I saw this person’s website @ <a href="https://zevoidah.github.io/" rel="nofollow noopener">https://zevoidah.github.io/</a></p>
<p>I looked at the code, but it shows that &lt;title&gt;𝐙𝐞 𝐕𝐨𝐢𝐝𝐚𝐡&lt;/title&gt; </p>
<p>How do you get the bolded text?</p>

**Comment by Socialdarwinist** [comment, marked as answer]
<p>Hello <a class="mention" href="/yangciou">@yangciou</a>, the title is not actually bold. Instead, the author of that page uses Unicode hacks – not the normal Latin characters, but characters from the <em>Mathematical Alphanumeric Symbols</em> block. For example he uses U+1D419 MATHEMATICAL BOLD CAPITAL Z instead of U+005A LATIN CAPITAL LETTER Z which is the usual letter Z.</p>
<p>You should use an application like <a href="https://wiki.gnome.org/Apps/Gucharmap" rel="nofollow noopener">gucharmap</a> to browse characters.</p>
<p>But don’t use letters from that block like normal letters. It makes your pages harder to find for machines. Use normal letters.</p>

**Comment by oycyc** [comment]
<p>Oh okay, thanks for the information!</p>
